tests/shells/remotemodules/test_tokenreader.py

```diff
@@ -5,11 +5,15 @@
 from crl.interactivesessions.shells.remotemodules.tokenreader import (
     TokenReader,
     SingleGapMatcher)
+from crl.interactivesessions.shells.remotemodules.compatibility import (
+    RANGE,
```

Comment:
Instead of RANGE from compatibility we could use builtins range. But this is quite okay like this as well.